recipes/xtensor/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "xtensor" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "025709893b89229f7723fd5c36faaaf8155c2d6558d63c2c51e9de1904f01c42" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/QuantStack/xtensor/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+test:
+  requires:
+    - gtest
+    - cmake
+
+  commands:
+    - test -d ${PREFIX}/include/xtensor  # [unix]
+    - test -f ${PREFIX}/include/xtensor/xarray.hpp  # [unix]
+    - if exist %LIBRARY_INC%\xtensor\xarray.hpp (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://quantstack.net/xtensor
+  license: BSD
+  license_file: LICENSE
+  summary: 'The C++ tensor algebra library'
+  description: 'Multi dimensional arrays with broadcasting and lazy computing'
+  doc_url: http://xtensor.readthedocs.io
+  dev_url: https://github.com/QuantStack/xtensor
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - jmabille
```

Comment:
So I think we missed checking with @jmabille . However, that handle doesn't actually point to any user. Who was this suppose to be? For now, I think I will take this handle off so that whoever this may be can be added in the feedstock.